File: threedollar-api-user-service/src/integrationTest/java/com/depromeet/threedollar/api/userservice/IntegrationTest.java
package com.depromeet.threedollar.api.userservice;

import com.depromeet.threedollar.domain.rds.domain.lib.DatabaseCleaner;
import org.junit.jupiter.api.AfterEach;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
public abstract class IntegrationTest {

    @Autowired
    private DatabaseCleaner databaseCleaner;

    @AfterEach
    void cleanup() {
        databaseCleaner.cleanUp();
    }

}

File: threedollar-api-boss-service/src/integrationTest/kotlin/com/depromeet/threedollar/api/bossservice/IntegrationTest.kt
package com.depromeet.threedollar.api.bossservice

import com.depromeet.threedollar.domain.rds.domain.lib.DatabaseCleaner
import org.junit.jupiter.api.AfterEach
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.TestConstructor

@TestConstructor(autowireMode = TestConstructor.AutowireMode.ALL)
@SpringBootTest
internal abstract class IntegrationTest {

    @Autowired
    private lateinit var databaseCleaner: DatabaseCleaner

    @AfterEach
    fun cleanup() {
        databaseCleaner.cleanUp()
    }

}
